cmd/expiration-mailer/main.go

```diff
@@ -54,6 +54,8 @@ type mailer struct {
 	nagTimes        []time.Duration
 	parallelSends   uint
 	limit           int
+	// Maximum number of rows to update in a single SQL UPDATE statement.
+	updateChunkSize int
```

Comment:
Since we know the max parameters that MariaDB will accept in a single prepared statement we should probably check the value of this field and refuse to start the expiration-mailer if it's exceeded.